test/base/src/androidTest/kotlin/io/realm/test/shared/RealmListTests.kt

```diff
@@ -293,6 +296,25 @@ class RealmListTests {
         assertEquals(2, realm.query<RealmListContainer>().find().size)
     }
 
+    @Test
+    fun listNotifications() = runBlocking {
+        val container = realm.writeBlocking { copyToRealm(RealmListContainer()) }
+        val collect = async {
+            container.objectListField.asFlow()
+                .takeWhile { it.list.size < 5 }
```

Comment:
This looks wrong? It will take the empty initial list and then complete? Shouldn't this be `it.list.size == 1` ?

Reply:
`takeWhile` collects until the condition is no longer met at which time the flow will be completed!? 

So we will keep on adding elements until there are five elements in the list and stop the test. ??

Reply:
Ah yeah, I was reading the test wrong 👍